yarn.lock

```diff
@@ -2423,9 +2423,9 @@ heroku-ps-exec@2.2.1:
     heroku-cli-util "^6.0.15"
     heroku-exec-util "0.6.2"
 
-heroku-redis@1.2.15:
-  version "1.2.15"
-  resolved "http://cli-npm.heroku.com/heroku-redis/-/heroku-redis-1.2.15/b1c58800efdcafdb100fc2418d12e260b5ea247d.tgz#b1c58800efdcafdb100fc2418d12e260b5ea247d"
+heroku-redis@1.3.0:
+  version "1.3.0"
+  resolved "https://registry.yarnpkg.com/heroku-redis/-/heroku-redis-1.3.0.tgz#738331c34311d7f872e9f52332363e8811bd108b"
```

Comment:
@RasPhilCo is the hostname change here expected?

Reply:
This is ok for now.

Reply:
Is this something in my local setup that I should do something about in future dep bump pull requests?